data/data.py

```diff
@@ -29,8 +29,35 @@ def check_hashes(d):
     print("There are " + str(counter) + " correct files.")
     return all_good
 
+def make_hash_list(directory, title):
```

Comment:
Ah yeah, that was just the function I made to generate the hash list and I thought it would be easiest to leave that in just in case we needed to generate another hash list quickly for whatever reason. I'll just leave it commented out since we don't need to call it in any script